app/src/main/java/com/example/mystore/ApiHelper.kt

```diff
@@ -1,7 +1,8 @@
 package com.example.mystore
 
-import com.example.mystore.models.ProductModel
+import com.example.mystore.model.ProductModel
+import retrofit2.Response
 
 interface ApiHelper {
```

Comment:
Naming is a little bit confusing, I was under the impression this was related to the networking layer but it is related to the repositories, consider renaming it with a more appropriate name.